src/vsr/replica_test.zig

```diff
@@ -1785,6 +1785,7 @@ test "Cluster: view_change: lagging replica repairs WAL using start_view from po
     a0.pass_all(.R_, .bidirectional);
     b1.pass(.R_, .outgoing, .do_view_change);
 
+    t.run();
```

Comment:
sic, I needed to add one more `.run` on top of what https://github.com/tigerbeetle/tigerbeetle/pull/2582 did